pilot/pkg/serviceregistry/kube/controller/ambient/authorization.go

```diff
@@ -25,6 +25,8 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pkg/config/schema/kind"
 	"istio.io/istio/pkg/log"
+	"istio.io/istio/pkg/maps"
+	"istio.io/istio/pkg/slices"
```

Comment:
Noticed this wasn't needed in the corresponding 1.24 backport, #54616 - is there any other relevant divergence that should be backported to 1.23 too?

Reply:
It's there: https://github.com/istio/istio/pull/54616/files#diff-5a6209247b2b9eadf04b73f00ce5513e147b01950b08d974240a4f05650732c3R28. Slices is just already imported in that branch because we have status reporting in 1.24 - https://github.com/keithmattix/istio/blob/f28c58c066fdfad0b0fa3197e8611893a6c9e258/pilot/pkg/serviceregistry/kube/controller/ambient/authorization.go#L452